egs/iban/s5/local/prepare_lm.sh

```diff
@@ -10,7 +10,7 @@ set -e -o pipefail
 
 local/train_lms_srilm.sh --train-text data/train/text data/ data/srilm
 
-nl -nrz -w10  iban/LM/iban-bp-2012.txt | sort -R > data/local/external_text
+nl -nrz -w10  corpus/LM/iban-bp-2012.txt | sort -R > data/local/external_text
```

Comment:
the database is being downloaded to corpus directory. Using a variable for
$corpus is not necessary.

On Mon, May 2, 2016 at 4:26 PM, jtrmal <notifications@github.com> wrote:

> In egs/iban/s5/local/prepare_lm.sh
> <https://github.com/kaldi-asr/kaldi/pull/744#discussion_r61797923>:
>
> > @@ -10,7 +10,7 @@ set -e -o pipefail
> >
> >  local/train_lms_srilm.sh --train-text data/train/text data/ data/srilm
> >
> > -nl -nrz -w10  iban/LM/iban-bp-2012.txt | sort -R > data/local/external_text
> > +nl -nrz -w10  corpus/LM/iban-bp-2012.txt | sort -R > data/local/external_text
>
> This should probably be $corpus or something like that
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/744/files/9a2c52ded3b2ed10d528d997e60727d67b4511db#r61797923>
>